transport/src/main/java/io/netty/bootstrap/Bootstrap.java

```diff
@@ -315,4 +316,9 @@ final SocketAddress remoteAddress() {
     final AddressResolverGroup<?> resolver() {
         return resolver;
     }
+
+    @Override
+    protected io.netty.channel.ChannelFactory getDefaultChannelFactory() {
```

Comment:
do we need the FQCN?

Reply:
@johnou yes, otherwise we are hitting a deprecated method